libs/core-components/src/lib/DropdownSelector/DropdownSelector.module.css

```diff
@@ -9,7 +11,7 @@
 */
 .container {
   /* Load select-css after Bootstrap overrides, because Bootstrap is like our "base" */
-  composes: form-control from '../../../styles/components/bootstrap.form.css';
+  composes: form-control from '@tacc/core-styles/src/lib/_imports/components/bootstrap.form.css';
```

Comment:
Should this be from /src/lib/_imports or from /dist? It appears inconsistently in other places.

Reply:
Yes, thanks.

- Updated this instance in 9acd951.
- Updated all other instances, but one[^1], in c237a15.

[^1]: I did **not** update the instance at [`SectionContent.layouts.module.css`](https://github.com/TACC/tup-ui/blob/task/TUP-280--ui-patterns--use-core-styles/libs/core-components/src/lib/SectionContent/SectionContent.layouts.module.css#L1-L2) because the `dist/` version of the import has no content (because (a) those tools/media-queries are not available in native CSS _yet_, and (b) core-styles does not _yet_ export source css to `dist/` files that would be empty).

Reply:
The straggling user of source is a known bother. Using PostCSS for now until I see another way.

---

**Maybe**, if source css cannot be compiled, then I just add it to dist **not** compiled. Not really "dist" then, is it? Well, I am __dist__-ributing it, so it fits in dist... but I should distinguish that it is source... maybe I name such files `dist/.../thing.src.css` or `dist/../thing.pcss` (`.pcss` is a standard). Hrm… I think I like this!

@jchuahtacc, thoughts?

Reply:
It should definitely be compiled if it's going out as dist, much in the same way that our distributed libraries for core-components go out as one umd file. It would also be nice to output one css file (i've seen an option *somewhere* where either vite or postcss outputs one file).